shared/cloud-gateway/overview/_product-overview.mdx

```diff
@@ -0,0 +1,68 @@
+import * as data from '@site/data/variables';
+
+<PlatformWrapper platform="linux">
+Deployed on your server, the Agora Cloud Gateway communicates with clients integrated with the [Agora RTC SDK](../reference/glossary#agora-video-sdk). You can use the Cloud Gateway to transmit video and audio streams between the server and the client through [SD-RTN™](../reference/glossary#agora-sd-rtn).
+
+The following diagram shows typical application scenarios of the Cloud Gateway, including playing local media files, pushing streams to CDN, and communicating with enterprise call centers. The Cloud Gateway implements media stream transmission by the following means:
+
+- Converting incoming media streams from SD&#8209;RTN™ to a specified format.
+- Encoding and sending local media streams to SD&#8209;RTN™.
+
+![1658370187632](https://web-cdn.agora.io/docs-files/1658370187632)
+
+<div class="alert note">When you use the <a href="../reference/glossary#agora-video-sdk">Agora RTC SDK</a> to communicate with the Cloud Gateway, set the channel scenario of the RTC SDK to <code>LIVE_BROADCASTING</code>.</div>
+
+<div class="alert note">The Cloud Gateway does not support SIP transcoding.</div>
+
+## Advantages
+
+The Gateway Server SDK has the following advantages:
+
+| Feature  | Description                                                         |
+| :----- | :----------------------------------------------------------- |
+| Reliability   | Supports cluster deployment, dynamic scalability, and high availability.                         |
+| Smoothness  | Low-latency media transmission from the server to the client based on the packet loss reduction feature of SD-RTN™.  |
+| High concurrency | For a server with an 8-core, 2.5 GHz CPU and 16 GB RAM, you can simultaneously send video streams (320 × 240, 15 fps) to 1,000 channels.|
```

Comment:
Need to update content. See: https://docs.agora.io/en/server_gateway/server_gateway_overview?platform=Linux

Reply:
Fixed.